state_machine/applications/system-check/tests/burnwire_test.py

```diff
@@ -24,11 +24,11 @@ def user_test():
           "ground pin (GND) and burnwire pin on the -Z solar board.")
 
     # get user input for voltage level and duration
-    voltage = float(input(
-        "At what voltage do you want to run the burnwire IC? (up to 3.3V): "))
-    burn_time = int(input(
-        "For how long do you want to run the burnwire test? "))
-    print("Test starting in", str(wait_time), "seconds.")
+    voltage = abs(float(input(
```

Comment:
This is strange. I'm not sure when we'd accidentally type in -5 rather than 5. 
Also I think you're fine to just not worry about malformed inputs when it comes to numeric inputs. 
But I think this is explicitly introduces improper handling of a malformed input.